tests/test_detect.py

```diff
@@ -0,0 +1,33 @@
+from unittest import TestCase
+
+from i18n.detect import Detect
+
+
+class TestExtract(TestCase):
+    """
+    Tests functionality of i18n/detect.py
+    """
+    def test_detect_changes(self):
+        """
+        Asserts that the correct value is returned from detect_changes method
+        """
+        detect = Detect()
+        self.assertFalse(detect.detect_changes())
+
+        file_name = 'conf/locale/fake2/LC_MESSAGES/mako.po'
+        with open(file_name, 'r') as f:
+            data = f.read()
```

Comment:
Add some comments explaining that you're backing up the file. Better yet, it might be easier to copy the file and delete the copy once you're done.